docs/API/header.md

```diff
@@ -82,5 +84,5 @@ We wanted the Header to be as customisable as possible, so you are free to try d
 | centerComponent | none | configuration object for default component (text: string, ...props for React Native Text component) valid React Element | define your center component here |
 | rightComponent | none | configuration object for default component (icon: string, ...props for React Native Elements Icon component) or a valid React Element | define your right component here |
 | backgroundColor | none | string | sets backgroundColor of the parent component |
-| outerContainerStyles | none | object (style) | styling for outer container |
-| innerContainerStyles | none | object (style) | styling for inner container |
+| outerContainerStyles | position: 'absolute',<br>top: 0,<br>left: 0,<br>right: 0,<br>backgroundColor: '#fff',<br>borderBottomColor: '#f2f2f2',<br>borderBottomWidth: 1,<br>padding: 15,<br>height: 70 | object (style) | styling for outer container |
```

Comment:
It may be easier and clearer to instead use a link to the definition in the source